### PR TITLE
Fixes #26357 - Show library/type of HTTP Proxy request

### PR DESCRIPTION
**Pull request by sbernhard**

### CODE REVIEW

File: lib/foreman/http_proxy/rest_client_extension.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ module HttpProxy
     module RestClientExtension
       def proxy_uri
         if proxy_http_request?(@proxy, @uri.hostname, @uri.scheme)
-          log_proxied_request(http_proxy, @uri.hostname)
+          log_proxied_request("RextClient", http_proxy, @uri.hostname)
```
Comment (timogoebel) [inline lib/foreman/http_proxy/rest_client_extension.rb]: Typo: `RestClient`

**Comment by sbernhard** [comment]
Typo fixed. Thank you! Ready for review @timogoebel

**Review by timogoebel — APPROVED** [review]

**Review by lzap — APPROVED** [review]
LGTM

**Comment by lzap** [comment]
Thank you, useful bit of info.

**Pull request merged**